etna/collections/models.py

```diff
@@ -485,6 +485,14 @@ class Meta:
         verbose_name_plural = _("highlight gallery pages")
 
     content_panels = BasePageWithIntro.content_panels + [
+        MultiFieldPanel(
+            heading="Content Warning Options",
+            classname="collapsible",
+            children=[
+                FieldPanel("display_content_warning"),
+                FieldPanel("custom_warning_text"),
```

Comment:
Hey @jamesbiggs, 

This looks good. I think if we're going to allow custom warning text, we either need to validate (in the clean method) that this has been set if an editor sets `display_content_warning` `True` or we need to supply a default to be used in the FE. From memory, we're wanting to set this individually?

Reply:
Hi Will, when we did this on the Articles pages, we put a default one in the HTML, which only shows if there's no `custom_warning_text`. I think we're following the same idea for this one here 😃